spec/classes/chrony_spec.rb

```diff
@@ -183,6 +184,7 @@
               it { is_expected.to contain_file('/etc/chrony.conf').with_content(%r{^\s*leapsecmode slew$}) }
               it { is_expected.to contain_file('/etc/chrony.conf').with_content(%r{^\s*leapsectz right/UTC$}) }
               it { is_expected.to contain_file('/etc/chrony.conf').with_content(%r{^\s*maxslewrate 1000\.0$}) }
+              it { is_expected.to contain_file('/etc/chrony.conf').with_content(%r{^\s*maxupdateskew 1000\.0$}) }
```

Comment:
Is there a reason for not adding a similar test for ArchLinux?

Reply:
No, I don't have a valid reason for that. I have to admin I blindly followed the code for `maxslewrate` ...

Reply:
I think we can refactor this part of the tests a bit because there is a lot of code duplication, but that's another story.  Can you update your branch to include the tests for Archlinux?

Reply:
Yes, https://github.com/voxpupuli/puppet-chrony/pull/112/commits/042d763c3bad098cd0ca9c4ac20f1bf8a87c2c47